pkg/api/flow.go

```diff
@@ -4432,6 +4442,159 @@ func (h *flowHandler) BroadcastCloudevent(w http.ResponseWriter, r *http.Request
 
 }
 
+func (h *flowHandler) BroadcastCloudeventFilter(w http.ResponseWriter, r *http.Request) {
+
+	h.logger.Debugf("Handling request: %s", this())
+
+	ctx := r.Context()
+	namespace := mux.Vars(r)["ns"]
+	filter := mux.Vars(r)["filter"]
+
+	ces, err := ToGRPCCloudEvents(r)
+	if err != nil {
+		respond(w, nil, err)
+		fmt.Println(err)
+		return
+	}
+
+	for i := range ces {
+
+		d, err := json.Marshal(ces[i])
+		if err != nil {
+			respond(w, nil, err)
+			fmt.Println(err)
+			return
+		}
+
+		inFilter := &grpc.ApplyCloudEventFilterRequest{
+			Namespace:  namespace,
+			Cloudevent: d,
+			FilterName: filter,
+		}
+
+		rsp, err := h.client.ApplyCloudEventFilter(ctx, inFilter)
+		if err != nil {
+			respond(w, nil, err)
+			fmt.Println(err)
+			return
+		}
+
+		if string(rsp.GetEvent()) == "null" {
+			respond(w, nil, nil) // drop event if not passed filter
+			fmt.Println("dropped event")
+			return
+		}
+
+		in := &grpc.BroadcastCloudeventRequest{
```

Comment:
Minor thing: CloudEvents is a proper noun, which should be capitalized the way I've done here.

Reply:
the function was already implemented should i change it ?

Reply:
Ah, no. In that case don't worry about it. It should have been capitalized the way I said, but I don't want to mess with backwards compatibility if it's been in there for a while.